components/ui/themes/DefaultTheme.ts

```diff
@@ -751,6 +751,11 @@ export const DefaultTheme = {
       borderTopLeftRadius: 0,
       borderTopRightRadius: 0,
     },
+    halfButton: {
+      borderRadius: 8,
```

Comment:
Do we need this half button styles anymore ?

Reply:
Yes, we are using this now